app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java

```diff
@@ -33,8 +33,12 @@
     private static enum BLUKON_STATES {
         INITIAL
     }
-    private static int m_getNowGlucoseDataIndexCommand = 0;
-    private static int m_gotOneTimeUnknownCmd = 0;
+    private static boolean m_getNowGlucoseDataIndexCommand = false;
+    private static boolean m_gotOneTimeUnknownCmd = false;
+    private static int GET_SENSOR_AGE_DELAY_MS =  3 * 3600;
```

Comment:
GET_SENSOR_AGE_DELAY_MS in milliseconds or seconds ?

Reply:
The only time it is used it is fed to `pratelimit` which takes a parameter in seconds. I think the field should be renamed `GET_SENSOR_AGE_DELAY` as the `MS` certainly looks confusing. The value looks correct for 3 hours.

Reply:
Yes it should be seconds :-) Will modify